docs/examples/extensions/examples.config.js

```diff
@@ -0,0 +1,104 @@
+/** @format */
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+const CopyWebpackPlugin = require( 'copy-webpack-plugin' );
+const fs = require( 'fs' );
+
+const extArg = process.argv.find( arg => arg.startsWith( '--ext=' ) );
+
+if ( ! extArg ) {
+	throw new Error( 'Please provide an extension.' );
+}
+
+const extension = extArg.slice( 6 );
+const extensionPath = path.join( __dirname, `${ extension }/js/index.js` );
+
+if ( ! fs.existsSync( extensionPath ) ) {
+	throw new Error( 'Extension example does not exist.' );
+}
+
+const externals = {
```

Comment:
Since these seem to be mostly shared with our main webpack config, maybe we can extract this list to another file and import to both configs.

Reply:
Absolutely! I was having trouble for some reason, so I'd rather tackle this in the next PR along with some other clean up.